agent.py

```diff
@@ -665,6 +669,53 @@ def save(self, stem):
         png.from_array(self.canvas, 'RGB').save('{}.png'.format(stem))
 
 
+class Draft:
+    RADIANT_RANGE = [0,1,2,3,4]
```

Comment:
Does this all still work if there's a human playing? Because humans and observers messes up the indices often